### PR TITLE
Feature/alcs 1874 tags on advanced search

### PR DESCRIPTION
**Pull request by fbarreta**
Advanced search for tags and categories.

### CODE REVIEW

File: services/apps/alcs/src/alcs/search/notice-of-intent/notice-of-intent-advanced-search.service.ts [hunk 1]
```diff
@@ -206,17 +183,14 @@ export class NoticeOfIntentAdvancedSearchService {
 
     const finalResult = intersectSets(allIds);
 
+    console.log(finalResult);
```
Comment (trslater) [inline services/apps/alcs/src/alcs/search/notice-of-intent/notice-of-intent-advanced-search.service.ts:186]: And this?

File: services/apps/alcs/src/alcs/search/notice-of-intent/notice-of-intent-advanced-search.service.ts [hunk 2]
```diff
@@ -197,6 +173,7 @@ export class NoticeOfIntentAdvancedSearchService {
 
     const allIds: Set<string>[] = [];
     for (const result of queryResults) {
+      console.log('count');
```
[the remark this answers is not in the document]
Comment (fbarreta) [reply services/apps/alcs/src/alcs/search/notice-of-intent/notice-of-intent-advanced-search.service.ts:176]: def not .... hahaha

File: alcs-frontend/.eslintrc.json [hunk 3]
```diff
@@ -1,6 +1,7 @@
 {
   "root": true,
   "ignorePatterns": ["projects/**/*"],
+  "no-octal-escape": 0,
```
Comment (Abradat) [inline alcs-frontend/.eslintrc.json:4]: I would prefer we move all the eslintrc changes to another since it is out of scope of this change

**Review by trslater — APPROVED** [review]
LGTM 🚀

File: alcs-frontend/src/app/features/search/search.component.html [hunk 4]
```diff
@@ -79,6 +79,65 @@ <h4>File Details</h4>
...
+                #tagInput
+                [matChipInputFor]="chipGrid"
+                [formControl]="tagControl"
+                [matAutocomplete]="auto"
+                [matChipInputSeparatorKeyCodes]="separatorKeysCodes"
+                (focus)="autoCompleteTrigger.openPanel()"
+                (click)="clearSearch()"
+                (blur)="checkDirty()"
+              />
+              @if (tags.length > 0) {
```
Comment (Abradat) [inline alcs-frontend/src/app/features/search/search.component.html:105]: You're using *ngFor in the bottom and @if here which is the new template syntax. Was there any specific reason for it?

**Pull request merged**